pokemongo_bot/step_walker.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, bot, speed, dest_lat, dest_lng):
 
     def step(self):
         if (self.dLat == 0 and self.dLng == 0) or self.dist < self.speed:
-            self.api.set_position(self.destLat, self.destLng, 0)
+            self.api.set_position(self.destLat + random_lat_long_delta(), self.destLng + random_lat_long_delta(), 0)
```

Comment:
utils.py

I figured it would be better if we didn't move to the exact stop coord.